packages/kbn-doc-links/src/get_doc_links.ts

```diff
@@ -52,17 +52,27 @@ export const getDocLinks = ({ kibanaBranch, buildFlavor }: GetDocLinkOptions): D
     },
     apm: {
       kibanaSettings: `${KIBANA_DOCS}apm-settings-in-kibana.html`,
-      supportedServiceMaps: `${KIBANA_DOCS}service-maps.html#service-maps-supported`,
-      customLinks: `${KIBANA_DOCS}custom-links.html`,
+      supportedServiceMaps: isServerless
+        ? `${SERVERLESS_DOCS}apm-service-map`
+        : `${KIBANA_DOCS}service-maps.html#service-maps-supported`,
```

Comment:
What's the context of this link? I'm guessing they should either both point to the "Supported APM agents" section or both point to the general page.

If the answer is both of them pointing to the "Supported APM agents" section, the serverless link should be updated to:

```
? `${SERVERLESS_DOCS}apm-service-map#supported-apm-agents`
```

Reply:
confirmed in code that this should point to supported service maps

```suggestion
        ? `${SERVERLESS_DOCS}apm-service-map #supported-apm-agents `
        : `${KIBANA_DOCS}service-maps.html#service-maps-supported`,
```